go.mod

```diff
@@ -46,7 +46,6 @@ require (
 	github.com/prometheus/procfs v0.0.7 // indirect
 	github.com/russross/blackfriday v2.0.0+incompatible // indirect
 	github.com/securego/gosec v0.0.0-20191119104125-df484bfa9e9f // indirect
-	github.com/soheilhy/cmux v0.1.4
```

Comment:
nit: Could run `go mod tidy` to drop cmux from go.sum as well.

Reply:
go mod tidy doesn't produce any changes

Reply:
That's surprising, I'd have thought it'd drop it from go.sum when it's no longer used.